src/core/pdg_generator/src/IntegrationWithSVF.cpp

```diff
@@ -74,7 +74,6 @@ bool NoelleSVFIntegration::runOnModule(Module &M) {
   // Pointer Analysis
   SVF::Options::PASelected.parseAndSetValue("nander");
   SVF::Options::PASelected.parseAndSetValue("sander");
-  SVF::Options::PASelected.parseAndSetValue("sfrander");
```

Comment:
Can you please comment this line rather than deleting it? 
Also, please add a comment saying why sfrander has been (hopefully temporarily) disabled.